dnd5e/module/actor/entity.d.ts

```diff
@@ -331,7 +331,7 @@ declare namespace Actor5e {
     newDay: boolean;
   };
 
-  interface Data extends Actor.Data<Data.Data, Item5e.Data> {
+  interface Data extends Actor.Data<Data.Data, Item5e['data']> {
```

Comment:
Do we need to let Actor5e.Data specify the Item5e.Data type?
e.g. something like `interface Data<ItemData extends Item.Data = Item5e.Data> extends Actor.Data<Data.Data, ItemData>` or a more generic `Item.Data` type

Reply:
I think we have to/should specify this will be `Item5e.Data`.